modules/base/src/processors/noiseprocessor.cpp

```diff
@@ -59,7 +59,7 @@ NoiseProcessor::NoiseProcessor()
     , levels_("levels", "Levels", 2, 8, 1, 16)
     , persistence_("persistence", "Persistence", 0.5f, 0.001f, 1.0f, 0.001f)
     , poissonDotsAlongX_("poissonDotsAlongX", "Dots Along X", 100, 1, 1024)
-    , poissonMaxPoints_("poissonMaxPoints", "Max Points", 1, 10000000, 10000000)
+    , poissonMaxPoints_("poissonMaxPoints", "Max Points", 1, 1, 10000000)
```

Comment:
should be `poissonMaxPoints_("poissonMaxPoints", "Max Points", 10000000, 1, 10000000)`